qcelemental/models/molecule.py

```diff
@@ -951,6 +951,10 @@ def from_file(cls, filename: str, dtype: Optional[str] = None, *, orient: bool =
             with open(filename, "rb") as infile_bytes:
                 data = deserialize(infile_bytes.read(), encoding="msgpack")
             dtype = "dict"
+        elif dtype == "msgpack-ext":
+            with open(filename, "rb") as infile_bytes:
+                data = deserialize(infile_bytes.read(), encoding="msgpack-ext")
+            dtype = "dict"
```

Comment:
since for reading json=json-ext and msgpack=msgpack-ext, aren't only 2 if's needed?

Reply:
Yes, although we should make it explicit. Ie, for json, use `json-ext` and for `msgpack`, use `msgpack-ext` (with comment that the `-ext` are backwards compatible)